### PR TITLE
Remove prefix on permissions

### PR DESCRIPTION
**Pull request by bastian-src**
According to https://github.com/theforeman/foreman_ansible/blob/master/lib/foreman_ansible/register.rb#L88

### CODE REVIEW

**Comment by dosas** [comment]
@bastian-src Any news on this? I would like to adapt the robottelo permission tests for this plugin but for this I require a decision about this PR.

**Comment by bastian-src** [comment]
Coming back to this, as far as I can tell, we should apply the suggested changes and remove the prefixes.

As a second opinion, @Thorben-D do you agree too?

As a reference, see [foreman_ansible](https://github.com/theforeman/foreman_ansible/blob/master/lib/foreman_ansible/register.rb#L88) and [foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution/blob/master/lib/foreman_remote_execution/engine.rb#L152).

**Comment by bastian-src** [comment]
TLDR;

After a discussion with @Thorben-D, we came to the conclusion that we keep the current permission and route definition.

---

The permissions relate to routes and also the usage of modules. Other plugins like `foreman_remote_execution`, `foreman_ansible`, and `foreman_rh_cloud` use these differently. For example, `foreman_remote_execution` encapsulates only their concerns in an extra ForemanRemoteExecution module:

<details>
<summary>Click to expand</summary>

```
tree app/models/
app/models/
├── concerns
│   ├── api
│   │   └── v2
│   │       ├── hosts_controller_extensions.rb
│   │       └── interfaces_controller_extensions.rb
│   └── foreman_remote_execution
│       ├── bookmark_extensions.rb
│       ├── errors_flattener.rb
...
│       ├── template_overrides.rb
│       └── user_extensions.rb
├── foreign_input_set.rb
├── host_proxy_invocation.rb
├── host_status
│   └── execution_status.rb
├── input_template_renderer.rb
├── invocation_provider_input_value.rb
...
```
</details>

`forema_salt` encapsulates everything in a ForemanSalt module, but orders the concerns differently:

<details>
<summary>Click to expand</summary>

```
tree app/models/
app/models/
└── foreman_salt
    ├── concerns
    │   ├── hostgroup_extensions.rb
    │   └── host_managed_extensions.rb
    ├── hostgroup_salt_module.rb
    ├── host_salt_module.rb
    ├── salt_environment.rb
...
```
</details>

`foreman_rh_cloud` does not even encapsulate its concerns:

<details>
<summary>Click to expand</summary>

```
tree app/models/
app/models/
├── concerns
│   └── rh_cloud_host.rb
├── insights_client_report_status.rb
├── insights_facet.rb
├── insights_hit.rb
...
```
</details>

And `foreman_puppet` encapsulates everything (maybe due to its history being part of foreman core) - but adds an additional `Extension` module:

<details>
<summary>Click to expand</summary>

```
app/models/
├── concerns
│   └── foreman_puppet
│       ├── extensions
│       │   ├── host_common.rb
│       │   ├── hostgroup.rb
...
│       │   └── user.rb
│       └── puppet_lookup_value_extensions.rb
└── foreman_puppet
    ├── config_group_class.rb
    ├── config_group.rb
    ├── environment_class.rb
...
```
</details>

In Resource Quota, we argued that Foreman Puppet's approach is the cleanest as it omits any constant naming conflicts. Therefore, we also stick to their route/permission definition which makes this PR obsolete.
